### PR TITLE
fix(install.sh) to work on GC Shell

### PR DESCRIPTION
**Pull request by rimusz**
as curl on Debian (Google Cloud Shell) does not download files from the Bintray

### CODE REVIEW

**Comment by sgoings** [comment]
Do they just not have curl installed?

**Comment by rimusz** [comment]
they do, but curls does not download files from bintray, very strange issue

**Comment by bacongobbler** [comment]
related: deis/deis.io#147

File: _scripts/installer/install.sh [hunk 1]
```diff
@@ -61,7 +61,11 @@ VERSION="$(get_latest_version "${HELM_VERSION_URL}")"
 HELM_ZIP="helm-${VERSION}-${PLATFORM}-${ARCH}.zip"
 
 echo "Downloading ${HELM_ZIP} from Bintray..."
-curl -Ls "${HELM_BIN_URL_BASE}/${HELM_ZIP}" -O
+if [[ $(which wget) ]]; then
```
Comment (sgoings) [inline _scripts/installer/install.sh]: This isn't quite the logic you want... you want something like:

`if command -v wget &> /dev/null; then`

This way you're specifically dealing with exit codes. See: http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script

**Comment by technosophos** [comment]
I don't quite understand what the issue is here. The code change looks fine to me, but I hate enabling insecure mode if we don't need to.

**Comment by rimusz** [comment]
In the worse scenario I can extra scripts (helm/deis) just for GC Shell use and we cam leave the current ones as it is

**Comment by sgoings** [comment]
Insecure `curl | bash` is definitely a no-no. We'll need to figure out how to at least do a secure `curl | bash` or handle this situation another way.

**Comment by rimusz** [comment]
closing this one, will make another script just for GC shell

**Pull request closed without merging**